extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -212,6 +212,17 @@ export class AmpAutocomplete extends AMP.BaseElement {
 
     /** @private {boolean} */
     this.interacted_ = false;
+
+    /**
+     * To ensure that we provide an accessible experience,
+    * the suggestion container must have a unique ID.
+     * In case the autocomplete doesn't have an ID we use a
+     * random number to ensure uniqueness.
+     @private {number|string} 
+     */
+    this.containerId_ = element.id
+      ? element.id
+      : Math.floor(Math.random() * 100) + '_AMP_content_';
```

Comment:
Is there a utility that generates ids similar to this elsewhere?

Reply:
IDTS, though I copied this almost directly from `amp-accordion`. It may be worth putting this in a util in a separate PR. Looks like `amp-mega-menu` uses this also.